src/main/java/com/org/ita/kata/implementation/MyroslavaVynokur/EigthImpl.java

```
@@ -9,8 +9,11 @@ public int liters(double time) {
     }
 
     @Override
-    public double getVolumeOfCuboid(double length, double width, double height) {
-        return 0;
+    public double getVolumeOfCuboid(double length, double width, double height) throws ArithmeticException {
```

need to throw the error above?

---

removed error